cockatrice/resources/help/search.md

```diff
@@ -33,7 +33,8 @@ The search bar recognizes a set of special commands similar to some other card d
 <dt><u>Pow</u>er, <u>Tou</u>ghness, <u>C</u>onverted <u>M</u>ana <u>C</u>ost:</dt>
```

Comment:
```suggestion
<dt><u>Pow</u>er, <u>Tou</u>ghness, <u>M</u>ana <u>V</u>alue:</dt>
```

Reply:
I'm not sure I agree, as we support both. Maybe include Mana Value in this, but leave CMC too

Reply:
It's just the headline of the group and doesn't alter or remove functionality.
In other groups (e.g. format or edition) we do also not list all possible synonyms and options:
![cmc](https://user-images.githubusercontent.com/9874850/124171433-deaa5d80-daa8-11eb-94f2-980838c2ade7.png)

I suggested to rename it here as that's the new default wording replacing cmc.
It's just a matter of time until the mana value term will be more popular and better known.

Reply:
mana value should now be the default option with cmc as a backup, we can safely remove cmc from the help file

Reply:
>mana value should now be the default option with cmc as a backup, we can safely remove cmc from the help file

I would not remove all notes of it right away as people might not all be aware or used to yet.
What do you think about removing it from the help file completely after the next main release? That should give our users enough transition time @ebbit1q.

Reply:
mana value was chosen because it is more intuitive, I don't think a user would be confused by the change as it converts this "CMC" to normal english

Reply:
>>>we can safely remove cmc from the help file
>>
>>What do you think about removing it from the help file completely after the next main release?
>>That should give our users enough transition time.
>
>I don't think a user would be confused by the change

Sure, accepting both code suggestions should do it then and remove CMC from the help file completely.